docs/integrations/apps/unified-billing/index.mdx

```diff
@@ -14,4 +20,19 @@ The following scopes pertain directly to Unified Partner Billing. For a complete
 | Subscriptions | modify | `account_subscriptions_cancel` | View or cancel account-level subscriptions | [GraphQL Account API](/docs/graphql-account/users-overview) |
 | Subscriptions | read-only | `account_subscriptions_read` | View account-level subscriptions | [GraphQL Account API](/docs/graphql-account/users-overview) |
 | Checkouts | modify | `account_checkouts_create` | View or cancel account-level checkouts | [GraphQL Account API](/docs/graphql-account/users-overview) |
-| Checkouts | read-only | `account_checkouts_read` | View account-level checkouts | [GraphQL Account API](/docs/graphql-account/users-overview) |
\ No newline at end of file
+| Checkouts | read-only | `account_checkouts_read` | View account-level checkouts | [GraphQL Account API](/docs/graphql-account/users-overview) |
+
+
+### Gathering Information 
+
+Collect the following values to use later.
+
+| Value | Description | 
+|:--------|:-----------|
+| Authentication headers | To authenticate API requests, visit the [account center (Staging Zone)](https://login.staging.zone/login) and generate a new account-level API account. Note the X-Auth-Client and X-Auth-Token properties because you must include them in the request headers. You can't reaccess the credentials from this page, so save them somewhere safe. Use the following list of OAuth scopes to inform the scopes you select for the API account. | 
```

Comment:
Are we still sending people to staging to test their integration out?

Reply:
resolving conversation, since the links are now changed